### ISSUE TITLE
IndexOutOfBoundsException 

### ISSUE THREAD

**Issue by tlf30** [andrewauclair/ModernDocking#71]
I somehow got into a state that when I added, I had 1 panel docked at the bottom and the new panel docked at the top, but no panel docked in the center, yet it was rendering the center as an empty pane. It would not let me drop a panel into the center. I fixed it by undocking all panels, then it was happy again. It threw this error twice in the process, I believe it was when I was adding the panel.
```
Exception in thread "AWT-EventQueue-0" java.lang.IndexOutOfBoundsException: Index -1 out of bounds for length 1
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:100)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckIndex(Preconditions.java:106)
	at java.base/jdk.internal.util.Preconditions.checkIndex(Preconditions.java:302)
	at java.base/java.util.Objects.checkIndex(Objects.java:385)
	at java.base/java.util.ArrayList.get(ArrayList.java:427)
	at ModernDocking.internal.DockedTabbedPanel.getSelectedTabID(DockedTabbedPanel.java:231)
	at ModernDocking.layouts.DockingLayouts.tabbedPanelToNode(DockingLayouts.java:131)
	at ModernDocking.layouts.DockingLayouts.panelToNode(DockingLayouts.java:105)
	at ModernDocking.layouts.DockingLayouts.layoutFromRoot(DockingLayouts.java:85)
	at ModernDocking.DockingState.getWindowLayout(DockingState.java:64)
	at ModernDocking.DockingState.getApplicationLayout(DockingState.java:75)
	at ModernDocking.persist.AppState$1.actionPerformed(AppState.java:125)
	at java.desktop/javax.swing.Timer.fireActionPerformed(Timer.java:311)
	at java.desktop/javax.swing.Timer$DoPostEvent.run(Timer.java:243)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by andrewauclair** [comment]
Interesting. Looks like it had something setup as a tabbed pane and tried to persist it with no selected tab.

**Comment by andrewauclair** [comment]
This is the same issue and fix as my latest change to #54.

**Comment by andrewauclair** [comment]
Fixed in 0.7.2
